jobs/ci-run/integration/integrationtests.yml

```diff
@@ -67,6 +67,14 @@
           echo "BUILD_ARCH=amd64" > build.properties
       - inject:
           properties-file: build.properties
+      - multijob:
```

Comment:
wrong place - these arm64 tests should not be nested under `gating-integration-tests-amd64`
and they also need to be wired up under the `gating-tests` job
and if we are adding the new bash tests, we also then need to remove the `gating-functional-tests-arm64` block